integration-cli/docker_cli_run_test.go

```diff
@@ -2718,6 +2718,20 @@ func TestRunAllowPortRangeThroughExpose(t *testing.T) {
 	logDone("run - allow port range through --expose flag")
 }
 
+// test docker run expose a invalid port
+func TestRunExposePort(t *testing.T) {
+	runCmd := exec.Command(dockerBinary, "run", "--expose", "80000", "busybox")
+	_, _, err := runCommandWithOutput(runCmd)
```

Comment:
Sorry, it's often extremely helpful to get the actual output text here as part of the test's error message, and then we can do a `!strings.Contains(<some error string here>)` to make sure we go the error we thought we should.